python/runtime/pai/submitter.py

```diff
@@ -41,6 +41,26 @@
 tensorflow-datasets==3.0.0
 """
 
+XGB_REQUIREMENT = TF_REQUIREMENT + """
+xgboost==0.82
+sklearn2pmml==0.56.0
+"""
+
+
+def get_requirement(model_name):
+    """Get required python package according to estimator name
+
+    Args:
+        estimator: name of the model
+
+    Returns:
+        A string with multilines, each line is a requirement
+    """
+    if model_name.lower() == "xgboost":
```

Comment:
How about `model_name.lower().startswith("xgboost")`? The `model_name` may be in the form of `xgboost.gbtree`.